docs/docs/icechunk-python/version-control.md

```diff
@@ -103,141 +98,138 @@ Now that we've created a new snapshot, we can time-travel back to the previous s
 
     It's important to note that because the `zarr Store` is read-only, we need to pass `mode="r"` to the `zarr.open_group` function.
 
-```python
-session = repo.readonly_session(snapshot_id="BSHY7B1AGAPWQC14Q18G")
+```python exec="on" session="version" source="material-block" result="code"
+session = repo.readonly_session(snapshot_id=list(repo.ancestry(branch="main"))[1].id)
```

Comment:
Is there a nicer way to get the second to last commit? equivalent to `HEAD~1`?

Reply:
`next(next(repo.ancestry(branch="main")))`?